core/extraction/towns_fund_round_four.py

```diff
@@ -23,8 +24,7 @@ def ingest_round_four_data_towns_fund(df_ingest: pd.DataFrame) -> dict[str, pd.D
     """
 
     towns_fund_extracted = dict()
-    # TODO: SMD-171 - update submission details extraction to support Round 4, currently hardcoded as "Round 3"
-    towns_fund_extracted["Submission_Ref"] = r3.extract_submission_details(df_ingest["1 - Start Here"].iloc[4, 1])
+    towns_fund_extracted["Submission_Ref"] = common.get_submission_details(4)
```

Comment:
not sure if it's strictly necessary, but it might be useful to invoke this param with it's kwarg? So it's more clear from glancing at it why/what it is hard-coded for?
any thoughts?